### DISCUSSION TITLE
Multiselect datetime repeater

### DISCUSSION THREAD

**Discussion by danswiser** [filamentphp/filament#5878] [Help] [answered]
I'm interested in using the filament forms package to replace a time tracking tool I've made.

Is it possible to select multiple dates with the Date-time picker form field and can each date selected then become a line item in the repeater field?

**Comment by danharrin** [comment, marked as answer]
With our builtin date picker you can only select one date, but there are other date picker plugins that could probably support this, or you could build a custom field.

As for creating repeater items, yes you can do that using `$set()` in a closure
